keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/AllPortsCustomizer.kt

```diff
@@ -0,0 +1,23 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.netflix.spinnaker.keel.api.ec2.AllPorts
+import io.swagger.v3.core.converter.AnnotatedType
+import io.swagger.v3.core.converter.ModelConverter
+import io.swagger.v3.core.converter.ModelConverterContext
+import io.swagger.v3.oas.models.media.Schema
+import io.swagger.v3.oas.models.media.StringSchema
+import org.springframework.stereotype.Component
+
+@Component
+class AllPortsCustomizer : BaseModelConverter() {
+  override fun resolve(annotatedType: AnnotatedType, context: ModelConverterContext, chain: MutableIterator<ModelConverter>): Schema<*>? {
+    val baseType = annotatedType.rawClass
+    return if (baseType == AllPorts::class.java) {
+      StringSchema().apply {
+        enum = listOf("ALL")
+      }
+    } else {
+      super.resolve(annotatedType, context, chain)
+    }
+  }
+}
```

Comment:
I'd like to generify this for any `object` that is a member of a `sealed class`. The simplest way to do it would seem to be to add `@JsonValue` to the definition, but that annotation can't go on a class/object.